components/dialog/demo/dialog-container.js

```diff
@@ -0,0 +1,40 @@
+import '../../button/button.js';
+import '../dialog.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+
+class DialogContainer extends LitElement {
```

Comment:
As promised in #280, this demo component opens the dialog by binding to a property of its consumer.  @maboelsoudd2l , I've so far not been able to reproduce the overflow error you mentioned.  

Note, when opening a dialog in such fashion, one must be careful to properly manage the state of the consumer's property (one way binding) since the dialog may close in ways other than changing the consumer's property - specifically, the user clicking a `dialog-action` button, the abort `[x]` button in top right corner, and the user hitting the `[escape]` key.